config/routes.rb

```diff
@@ -22,6 +22,7 @@
 
   resources :users do
     resources :orders
+    match "/create_order", via: [:get], to: "orders#create"
```

Comment:
I'm not sure either.  I only did it to get the tests to pass.  Anything you want to do is fine with me.